.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+image: docker:latest
+stages:
+  - deploy
+services:
+  - docker:dind
+
+variables:
+  # REPLACE THIS, and set DOCKERHUB_TOKEN & AZURE_WEBHOOK in pipeline variables
+  # Make sure that "Expand Variable" is **off** for the webhook, and that both secrets are masked
+  DOCKERHUB_USERNAME: corndeldevopscourse
+
+deploy:
+  stage: deploy
+  # UNCOMMENT IF YOU WANT A BRANCH CHECK
+  # only:
+  #   refs:
+  #     - master
+
+  script:
+    - docker build --tag ${DOCKERHUB_USERNAME}/m13-order-processing-app --target production .
+    - echo $DOCKERHUB_TOKEN | docker login -u $DOCKERHUB_USERNAME --password-stdin
+    - docker push ${DOCKERHUB_USERNAME}/m13-order-processing-app
+    - apk add --update-cache curl
+    - curl -dHf -X POST "$AZURE_WEBHOOK"
```

Comment:
Responding to previous comment:
> Do we need any of the -dHf flags here? (-f seems actively harmful)

Good spot. Without `-f` this step can fail silently, but `--fail-with-body` would be better, and will remove `-dH`